dynamic_rest/serializers.py

```diff
@@ -83,6 +84,16 @@ def __new__(cls, *args, **kwargs):
             meta = type('Meta', (), {})
             cls.Meta = meta
         meta.list_serializer_class = DynamicListSerializer
+
+        # Check for bulk resource creation with help of resource plural name.
+        # This should happen before actual constructor because class needs to
+        # be changed on DynamicListSerializer.
+        plural_name = cls.get_plural_name()
+        data = kwargs.get('data', fields.empty)
+        if data is not fields.empty and plural_name in data and len(data) == 1:
```

Comment:
This is really cool -- if we're doing this here, do we even need the BulkCreateMixin? If we expand this check so that it looks for list-type values, we can make all DREST serializers support bulk creation:

```
if data is not fields.empty:
  if plural_name in data and len(data) == 1:
    kwargs['data'] = data[plural_name]
    kwargs['many'] = True
  elif isinstance(data, list):
    kwargs['many'] = True
```